hibernate-core/src/main/java/org/hibernate/tool/schema/spi/Exporter.java

```diff
@@ -22,13 +24,45 @@
 public interface Exporter<T extends Exportable> {
 	String[] NO_COMMANDS = new String[0];
 
+	/**
+	 * Get the commands needed for creation.
+	 *
+	 * @return The commands needed for creation scripting.
+	 * @deprecated Will be removed in favor of the variant accepting {@link SqlStringGenerationContext}
+	 * @see #getSqlCreateStrings(Exportable, Metadata, SqlStringGenerationContext)
+	 */
+	@Deprecated
+	default String[] getSqlCreateStrings(T exportable, Metadata metadata) {
```

Comment:
Just a warning: you're restoring backwards compatibility for callers of this method, but not for implementors. This method will never get called by Hibernate ORM.